lib/animina/user_email.ex

```diff
@@ -6,43 +6,38 @@ defmodule Animina.UserEmail do
   import Swoosh.Email
   import AniminaWeb.Gettext
   alias Animina.Mailer
-  use AshAuthentication.Sender
 
-  def send(user, confirm, _opts) do
+  def send_pin(name, email, pin) do
     subject = gettext("👫❤️ Confirm the email address for your new ANIMINA account")
 
     body =
-      construct_salutation(user) <>
-        construct_email_body() <>
-        construct_link(confirm) <>
+      construct_salutation(name) <>
+        construct_email_body(pin) <>
         construct_signature()
 
     send_email(
-      user.name,
-      Ash.CiString.value(user.email),
+      name,
+      Ash.CiString.value(email),
       subject,
       body
     )
   end
 
-  defp construct_salutation(user) do
-    "Hi #{user.name}!\n"
+  defp construct_salutation(name) do
+    "Hi #{name}!\n"
   end
 
-  defp construct_email_body do
-    ~S"""
-
-    Your new ANIMINA https://animina.de account has been created with this
+  defp construct_email_body(pin) do
+    "Your new ANIMINA https://animina.de account has been created with this
     email address.
 
-    Please confirm the email address by clicking the monstrosity of a link
-    below or do nothing in case you didn't create the account.
+    Please use this 6 digit PIN to verify your new ANIMINA account.
 
-    """
-  end
 
-  defp construct_link(confirm) do
-    "\n #{"#{get_link(Application.get_env(:animina, :env))}/auth/user/confirm_new_user?#{URI.encode_query(confirm: confirm)}"}"
+    #{pin}
```

Comment:
Yes , this is how it looks

<img width="1129" alt="Screenshot 2024-11-04 at 07 51 18" src="https://github.com/user-attachments/assets/7090722e-ff2d-444c-b9a0-e7554fe01e98">

It is easy to see the pin

Reply:
I'd prefer something like:

```text
Hi kamala!

Please use this PIN to verify your new ANIMINA account:

    315223

Best regards,
ANIMINA Team

-- 
https://animina.de
This email was sent by the ANIMINA system. Please do not reply to it.
I am just dumb software and can't read your emails. Do contact my human boss Stefan Wintermeyer in case you have any questions or
```